sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/CountIT.java

```diff
@@ -0,0 +1,91 @@
+package com.azure.spring.data.cosmos.repository.integration;
```

Comment:
New file header missing.

Reply:
i always forget that!  weird that one of the static analysis tools didn't flag that but good catch - header added